pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -109,10 +109,10 @@ func (p *prtbLifecycle) syncPRTB(binding *v3.ProjectRoleTemplateBinding) error {
 	if binding.UserName == "" && binding.GroupPrincipalName == "" && binding.GroupName == "" {
 		return nil
 	}
-
 	rt, err := p.rtLister.Get("", binding.RoleTemplateName)
 	if err != nil {
-		return fmt.Errorf("couldn't get role template %v: %w", binding.RoleTemplateName, err)
+		logrus.Warnf("ProjectRoleTemplateBinding %q sets a non-existing role template %q. Skipping.", binding.Name, binding.RoleTemplateName)
```

Comment:
I think we should differentiate between `NotFound` and other errors. You could use `apierrors.IsNotFound(err)` both here and in namespace handler

Reply:
Excellent point, thanks!